src/api.js

```diff
@@ -14,7 +14,7 @@ function open(file) {
 
 function rasterize(geojson, args) {
     return new Promise((resolve, reject) => {
-        resolve(new GDALDataset({ func: 'GDALRasterize', src: geojson, args: args }));
+        resolve(new GDALDataset(new DatasetSource('GDALRasterize', geojson, [], args)));
```

Comment:
Is this a breaking change? That which used to accept an object now expects a `DatasetSource`?

Reply:
Good question! This should not be a breaking change because `DatasetSource` has exactly the same shape as the object that this previously accepted, and passing in the object will still work. This was just for code organization purposes because I had trouble tracking down where `func` was coming from and wanted to make it more explicit. Eventually I'd like to switch to Typescript, which would make it easier to track these types of things.